sites/melbourne/lib/melbourne/data/llm_preparer.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Melbourne
+  module Data
+    class LLMPreparer
+      def initialize(path:)
+        @json_contents = JSON.parse(File.read(path))
+      end
+
+      def prepare # rubocop:disable Metrics/MethodLength
```

Comment:
Same as above; these classes are just data processing scripts not meant to live forever or to be used regularly at all.